storybook/main.qml

```diff
@@ -0,0 +1,65 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+
+import StatusQ.Core 0.1
+import StatusQ.Components 0.1
+import StatusQ.Layout 0.1
+
+import Qt.labs.settings 1.0
+
+
+ApplicationWindow {
+    id: rootWindow
+
+    width: 1450
+    height: 840
+    visible: true
+
+    title: qsTr("Status Desktop Storybook App")
```

Comment:
You don't need this; it's either taken from "applicationName" or "applicationDisplayName" (see main.cpp)